apps/files_versions/lib/Versions/VersionManager.php

```diff
@@ -133,4 +138,45 @@ public function deleteVersion(IVersion $version): void {
 			$backend->deleteVersion($version);
 		}
 	}
+
+	/**
+	 * Catch ManuallyLockedException and retry in app context if possible.
+	 *
+	 * The files_lock app may throw ManuallyLockedExceptions
+	 * when attempting to revert a file that is currently opened.
+	 * This would prevent the user from rolling back a opened file.
+	 *
+	 * Text and Richdocuments handle changes of the file while editing.
+	 * Allow reverting files even when they are locked by these apps
+	 * and let the apps handle the conflict.
```

Comment:
A bit nitpicky, but I find it cleared. I might be saying wrong things though.

```suggestion
	 * Context: The files_lock app will throw ManuallyLockedExceptions
	 * when writing to a file that is currently locked.
	 * This is to prevents a user from editing an already opened file.
	 *
	 * Exception: Text and Richdocuments can handle changes of the file while editing.
	 * So we execute the edition under their lock context
	 * to let them handle the conflict.
```

Reply:
Thanks for the proposal. It's really only about rolling back to old versions - not about editing. I will take your changes as a starting point and rephrase from there. :)

Reply:
I did rephrase it. Curious to hear what you think @artonge